src/components/drag-and-drop/custom-drag-layer/custom-drag-layer.js

```diff
@@ -62,6 +62,10 @@ class CustomDragLayer extends React.Component {
     };
   }
 
+  getKlassName = (props) => {
+    return `custom-drag-layer ${props.className}`;
```

Comment:
if `props.className` is not set, then this will add a class of `undefined` into the HTML.

Instead please use the [classNames](https://github.com/JedWatson/classnames) package we use in Carbon:

```
classNames('custom-drag-layer', props.className);
```